addons/l10n_rs_edi/models/account_move.py

```diff
@@ -0,0 +1,164 @@
+import uuid
+import requests
+import json
+
+from odoo import _, api, fields, models
+from requests.exceptions import Timeout, ConnectionError, HTTPError
+
+DEMO_EFAKTURA_URL = 'https://demoefaktura.mfin.gov.rs/api/publicApi/sales-invoice/ubl'
+EFAKTURA_URL = 'https://efakturadev.mfin.gov.rs/api/publicApi/sales-invoice/ubl'
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_rs_edi_uuid = fields.Char(
+        string="RS Invoice UUID",
+        compute="_compute_l10n_rs_edi_uuid",
+        copy=False,
+        store=True,
+        help="Unique Identifier for an invoice used as request id",
+    )
+
+    l10n_rs_edi_is_eligible = fields.Boolean(
+        compute="_compute_l10n_rs_edi_is_eligible",
+        store=True,
+        help="Technical field to determine if this invoice is eligible to be e-invoiced.",
+    )
+
+    l10n_rs_edi_attachment_file = fields.Binary(
+        string="Serbian E-Invoice XML File",
+        copy=False,
+        attachment=True,
+        help="Serbia: technical field holding the e-invoice XML data.",
+    )
+
+    l10n_rs_edi_attachment_id = fields.Many2one(
+        comodel_name='ir.attachment',
+        string="eFaktura XML Attachment",
+        compute=lambda self: self._compute_linked_attachment_id('l10n_rs_edi_attachment_id', 'l10n_rs_edi_attachment_file'),
+        depends=['l10n_rs_edi_attachment_file'],
+    )
```

Comment:
Ubl already has its own fields: ubl_cii_xml_id / ubl_cii_xml_file. Why do you need new ones?